sbt/src/sbt-test/tests/test-exclude/test

```diff
@@ -0,0 +1,55 @@
+# Test1 & Test2 create files Test1.run & Test2.run respectively
+
+# no parameters
+> test-only
+$ exists target/Test1.run
+$ exists target/Test2.run
+
+$ delete target/Test1.run
+$ delete target/Test2.run
+
+
+# with explicit match
+> test-only Test1*
+$ exists target/Test1.run
+-$ exists target/Test2.run
+
+$ delete target/Test1.run
+
+
+# with explicit match and exclusion
+> test-only Test* -Test1
+-$ exists target/Test1.run
+$ exists target/Test2.run
+
+$ delete target/Test2.run
+
+
+# with explicit match and exclusion
+> test-only Test* -Test2
+$ exists target/Test1.run
+-$ exists target/Test2.run
+
+$ delete target/Test1.run
+
+
+# with only exclusion
+> test-only -Test2
+$ exists target/Test1.run
+-$ exists target/Test2.run
+
+$ delete target/Test1.run
+
+
+# with only exclusion
+> test-only -Test1
+-$ exists target/Test1.run
+$ exists target/Test2.run
+
+$ delete target/Test2.run
+
+
+# with only glob exclusion
+> test-only -Test*
```

Comment:
Here, we should have an additional test with some arbitrary string, like -Blah to see if my intuitation about joining filters is correct.